ag_data/formulas/library/system/mercury_formulas.py

```diff
@@ -0,0 +1,55 @@
+#
+# Formulas for Mercury's built-in and natively supported sensor types
+# Sample processing formulas with primary sensors and fictional Mercury-branded sensors
+#
+
+from ag_data import models
+
+
+def fPass(timestamp, sensor, measurement_payload):
+    print("passing")
+
```

Comment:
Here, when we don't need any process for data, should we return measurement_payload instead of return None, so that the frontend team will be able to directly fetch data from measurement.value.get("result"), otherwise they still need to check measurement.value.get("reading").